test/option/fixture/SConstruct__experimental

```diff
@@ -0,0 +1,6 @@
+DefaultEnvironment(tools=[])
+env = Environment(tools=[])
+
+exp=GetOption('experimental')
+print("Experimental=%s"%exp)
+print("All=%s"%('all' in exp))
```

Comment:
perhaps run this new file through code formatter?

Reply:
will do.